change/@internal-react-composites-093c2064-8b62-442d-bf87-d075572716be.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
this should be minor as it visually impacts stable builds (even more so if contoso has set their own theme.effect.roundercorner2/4 value)